packages/docs/src/pages/patterns/overview.md

```diff
@@ -0,0 +1,15 @@
+---
+label: Overview
+title: Patterns
+weight: -1
+---
+Patterns are solutions and researched best practices that solve recurring user interface design problems:
+
+  * How do you build complex, multi-part web forms?
+  * How do you provide navigational context?
+  * How do you handle progressive disclosure?
+
+## How are they different from components?
+A pattern is more than the sum of its parts. Patterns are solutions, whereas a component can be considered a UI chunk. Patterns consist of design, content strategy, reusable components, and accessibility put together to solve common problems on CMS.gov.
```

Comment:
Suggest replacing `CMS.gov` with `CMS websites`

Reply:
Could we remove `on CMS.gov` and end the sentence after `problems` ?